Application crashes on startup

### Describe the bug:

Application crash 

### To Reproduce:

Compile and run hello world app:

```go
package main

import (
	"fyne.io/fyne/app"
	"fyne.io/fyne/widget"
)

func main() {
	a := app.New()

	w := a.NewWindow("Hello")
	w.SetContent(widget.NewVBox(
		widget.NewLabel("Hello Fyne!"),
		widget.NewButton("Quit", func() {
			a.Quit()
		}),
	))

	w.ShowAndRun()
}
```

### Trace:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x42456ac]

goroutine 1 [running, locked to thread]:
fyne.io/fyne.CurrentDevice(0x0, 0xc0000d7000)
	/Users/andrey/go/src/fyne.io/fyne/device.go:40 +0x2c
fyne.io/fyne/app.defaultTheme(0x446794c, 0xa)
	/Users/andrey/go/src/fyne.io/fyne/app/app_darwin.go:17 +0x22
fyne.io/fyne/app.(*settings).setupTheme(0xc000162100)
	/Users/andrey/go/src/fyne.io/fyne/app/settings.go:127 +0x9e
fyne.io/fyne/app.(*settings).load(0xc000162100)
	/Users/andrey/go/src/fyne.io/fyne/app/settings.go:91 +0xe2
fyne.io/fyne/app.loadSettings(...)
	/Users/andrey/go/src/fyne.io/fyne/app/settings.go:133
fyne.io/fyne/app.NewAppWithDriver(0x44f40e0, 0xc0000d7020, 0x0, 0x0, 0x4445901, 0xc0000c70a0)
	/Users/andrey/go/src/fyne.io/fyne/app/app.go:99 +0x61
fyne.io/fyne/app.NewWithID(...)
	/Users/andrey/go/src/fyne.io/fyne/app/app_gl.go:13
fyne.io/fyne/app.New(0xc0000c70a0, 0xc0000f9f58)
	/Users/andrey/go/src/fyne.io/fyne/app/app.go:86 +0x8b
main.main()
	/tmp/bug/main.go:9 +0x26
```

### Device (please complete the following information):
 - OS: MacOS 
 - Version: 10.13.3
 - Go version: go1.12.5 darwin/amd64
 - Fyne version: 7df22f0c81f45ed8abefaa0b790aa4f835eb9c95

--- a/app/app.go
+++ b/app/app.go
@@ -99,7 +99,7 @@ func NewAppWithDriver(d fyne.Driver, id string) fyne.App {
 	newApp := &fyneApp{uniqueID: id, driver: d,
 		prefs: prefs, exec: exec.Command}
 	fyne.SetCurrentApp(newApp)
-	newApp.settings = loadSettings();
+	newApp.settings = loadSettings()
 
 	listener := make(chan fyne.Settings)
 	newApp.Settings().AddChangeListener(listener)

--- a/app/app.go
+++ b/app/app.go
@@ -96,9 +96,10 @@ func NewAppWithDriver(d fyne.Driver, id string) fyne.App {
 	} else {
 		prefs = loadPreferences(id)
 	}
-	newApp := &fyneApp{uniqueID: id, driver: d, settings: loadSettings(),
+	newApp := &fyneApp{uniqueID: id, driver: d,
 		prefs: prefs, exec: exec.Command}
 	fyne.SetCurrentApp(newApp)
+	newApp.settings = loadSettings();
 
 	listener := make(chan fyne.Settings)
 	newApp.Settings().AddChangeListener(listener)

